src/help/guides/adding-repos.md

```diff
@@ -5,24 +5,36 @@ lang: en-US
+1. Choose a source to install.
 
 ## Known Repositories
 This is a list of all of the known repositories currently for Paperback, as well as the repository contents.
-| Repository Name | Base URL | Available Sources | Notes | Add Repo (iOS Only) |
-| :-------------: | :------: | :---------------- | :---: | :---:
-| **Primary Sources**   | https://paperback-ios.github.io/extensions-beta | <ExtensionsList url="https://paperback-ios.github.io/extensions-beta"/> | Officialy maintained repository, contains a few major sources suited for most users | [Add to Paperback](paperback://addRepo?displayName=Primary%20Sources&url=https://paperback-ios.github.io/extensions-beta) |
-| **H-Extensions**      | https://paperback-ios.github.io/h-extensions | <ExtensionsList url="https://paperback-ios.github.io/h-extensions"/> | Official repository for 18+ sources. These also include redirector versions of the source, which allow you to view content even if it is blocked in your country, without a VPN | [Add to Paperback](paperback://addRepo?displayName=H-Extensions&url=https://paperback-ios.github.io/h-extensions) |
-| **MangaDex Unlocked** | https://pogogo007.github.io/extensions-beta | <ExtensionsList url="https://pogogo007.github.io/extensions-beta"/> | This source overwrites MangaDex and unlocks the full contents of the website. No content is filtered. May load faster than the built-in version of the source | [Add to Paperback](paperback://addRepo?displayName=MangaDex%20Unlocked&url=https://pogogo007.github.io/extensions-beta) |
-| **Beta Extensions**   | https://conradweiser.github.io/extensions | <ExtensionsList url="https://conradweiser.github.io/extensions"/> | These extensions are upcoming to the official repository. These sources may or may not work | [Add to Paperback](paperback://addRepo?displayName=Beta%20Extensions&url=https://conradweiser.github.io/extensions) |
+
+<div>
+    <ExtensionsList
+        url="https://paperback-ios.github.io/extensions-beta"
+        name="Primary Sources"
+        description="Officially maintained repository, contains a few major sources suited for most users" />
+    <ExtensionsList 
+        url="https://paperback-ios.github.io/h-extensions"
+        name="H-Extensions"
+        description="Official repository for 18+ sources. These also include redirector versions of the source, which allow you to view content even if it is blocked in your country, without a VPN" />
+    <ExtensionsList 
+        url="https://pogogo007.github.io/extensions-beta"
+        name="MangaDex Unlocked"
+        description="This source overwrites MangaDex and unlocks the full contents of the website. No content is filtered. May load faster" />
+    <ExtensionsList 
+        url="https://conradweiser.github.io/extensions"
+        name="Beta Extensions"
+        description="These extensions are upcoming to the official repository. These sources may or may not work" />
```

Comment:
Until promises are used in the beta/stable version we should remove the beta repo
```suggestion
```